src/main/java/org/fcrepo/camel/RdfNamespaces.java

```diff
@@ -29,6 +33,20 @@
 
     public static final String LDP = "http://www.w3.org/ns/ldp#";
 
+    public static final String SERVER_MANAGED = REPOSITORY + "ServerManaged";
```

Comment:
None of the below properties are used outside of tests.
Do we need to keep the additions to this class?